docs/index.rst

```diff
@@ -48,6 +48,17 @@ This option is highly recommended:
 
    $ pip install aiodns
 
+Installing speedups altogether
+------------------------------
+
+The following will get you :term:`aiohttp` along with :term:`chardet`,
```

Comment:
```console
Warning, treated as error:
/home/travis/build/aio-libs/aiohttp/docs/index.rst:54:term not in glossary: aiohttp
```

@asvetlov should we add it there? how to deal with this error?

Reply:
Do you really want a link to glossary here?
Maybe just backticks?
 ```
``aiohttp``
```
Otherwise we need a glossary definition etc.